scripts/mk.js

```diff
@@ -14784,47 +14822,47 @@ var itemDistributions = {
 	}, {
 		name: toLanguage("Mushrooms", "Champis"),
 		value: [{
-			"champi": 1
+			"mushroom": 1
 		}, {
-			"champi": 1,
+			"mushroom": 1,
 			"poison": 1
 		}, {
-			"champi": 4,
+			"mushroom": 4,
 			"poison": 1,
-			"megachampi": 1
+			"megamushroom": 1
 		}, {
-			"champi": 3,
+			"mushroom": 3,
 			"poison": 1,
-			"megachampi": 1
+			"megamushroom": 1
 		}, {
-			"champi": 3,
+			"mushroom": 3,
 			"poison": 1,
-			"champiX3": 1,
-			"megachampi": 2
+			"mushroomX3": 1,
+			"megamushroom": 2
 		}, {
-			"champi": 2,
-			"champiX3": 1,
-			"megachampi": 2
+			"mushroom": 2,
+			"mushroomX3": 1,
+			"megamushroom": 2
 		}, {
-			"champi": 1,
-			"champiX3": 2,
-			"megachampi": 3
+			"mushroom": 1,
+			"mushroomX3": 2,
+			"megamushroom": 3
 		}, {
-			"champiX3": 2,
-			"megachampi": 2,
-			"champior": 1
+			"mushroomX3": 2,
+			"megamushroom": 2,
+			"goldMushroom": 1
 		}, {
-			"champiX3": 1,
-			"megachampi": 1,
-			"champior": 1
+			"mushroomX3": 1,
+			"megamushroom": 1,
+			"goldMushroom": 1
 		}, {
-			"champiX3": 1,
-			"champior": 1
+			"mushroomX3": 1,
+			"goldMushroom": 1
 		}, {
-			"champiX3": 1,
-			"champior": 2
+			"mushroomX3": 1,
+			"goldMushroom": 2
 		}, {
-			"champior": 1
+			"goldMushroom": 1
```

Comment:
This will require a migration... Users can have their own item distribution, the item names are mentioned on them